client/containers/Budget/index.js

```diff
@@ -25,9 +25,9 @@ class Budget extends Component {
     const data = { transactions, categories };
 
     return (
-      <div>
-        <BudgetGrid data={data} />
-      </div>
+      <Chunk load={() => import('components/BudgetGrid')}>
```

Comment:
Since this function is re-created on every render, it might be more efficient to move it outside the component and reference here